### PR TITLE
Add port configuration

### PR DESCRIPTION
**Pull request by rofrol**

### CODE REVIEW

File: docker-compose.dev.yml [hunk 1]
```diff
@@ -31,6 +31,9 @@ services:
...
       - ./fe/public/:/var/www/test1/
     ports:
-      - '80:80'
+      - '${WEB_SERVER_PORT}:${NGINX_PORT}'
     links:
-      - web:web
\ No newline at end of file
+      - web:web
+    environment:
+      - NGINX_PORT=${NGINX_PORT}
```
Comment (mariuszklinger) [inline docker-compose.dev.yml]: can we set default to `80` if `NGINX_PORT` is not set?

Comment (rofrol) [reply docker-compose.dev.yml]: We have this in `.env.example`

Comment (rofrol) [reply docker-compose.dev.yml]: So default port will be set. If someone does not do `cp .env.example .evn`, docker-compose will not even start.

Comment (rofrol) [reply docker-compose.dev.yml]: Added default port 80

**Pull request merged**